webpages/styles/components/smallmode.css

```
@@ -15,6 +15,11 @@
   cursor: pointer;
   width: 24px;
   height: 24px;
-  margin-inline-start: 13px;
-  margin-inline-end: 5px;
+  padding: 18px;
+  margin-inline-start: -14px;
```

```suggestion
  margin-inline-start: -5px;
```
This aligns the close icon with the category icons when the dropdown is opened.

---

it looks a bit weird tho...